sdl_android/src/main/java/com/smartdevicelink/transport/MultiplexBluetoothTransport.java

```diff
@@ -257,9 +254,9 @@ public synchronized void connected(BluetoothSocket socket, BluetoothDevice devic
         	mConnectedWriteThread = null;
         }
         // Cancel the accept thread because we only want to connect to one device
-        if (!keepSocketAlive && getBluetoothSerialServerInstance().mSecureAcceptThread != null) {
-        	getBluetoothSerialServerInstance().mSecureAcceptThread.cancel();
-            getBluetoothSerialServerInstance().mSecureAcceptThread = null;
+        if (!keepSocketAlive && mSecureAcceptThread != null) {
+        	mSecureAcceptThread.cancel();
+            mSecureAcceptThread = null;
         }
 
         // Start the thread to manage the connection and perform transmissions
```

Comment:
Actually I believe that it doesn't matter if the name equals `null` or empty set. It's just about setting the value here. So just `if(device !=null)` will suffice.